thoth/common/helpers.py

```diff
@@ -57,10 +59,24 @@ def datetime_str_from_timestamp(timestamp: int) -> str:
     return datetime2datetime_str(datetime.datetime.fromtimestamp(timestamp))
 
 
+def get_incluster_token_file(token_file=None):
```

Comment:
I would probably hide these functions by naming them `_get_incluster_token_file` and `_get_incluster_ca_file` so they are not exposed outside of module.